develop-docs/sdk/features.mdx

```diff
@@ -267,6 +267,10 @@ Add a breadcrumb for each outgoing HTTP request after the request finishes:
 
 - type: `http`
 - category: `http`
+- level
+  - `info` - response status code 2XX - 3XX
+  - `warning` - response status code 4XX
+  - `error` - response status code 5XX
```

Comment:
I am not totally sold on 4xx being a warning. 4xx is equally as bad from the client perspective as 5xx. (if not worse, because you are actually on the client)